src/pages/home/component/carousel/Carousel.jsx

```diff
@@ -0,0 +1,50 @@
+import "./carousel.css";
+import { CarouselData } from "./carousel-data";
+import { useEffect, useState, useRef } from "react";
+import ArrowLeftIcon from "@material-ui/icons/ArrowLeft";
+import ArrowRightIcon from "@material-ui/icons/ArrowRight";
+
+const Carousel = () => {
+    const carouselRef = useRef(null);
+    const [currentImageIndex, setCurrentImageIndex] = useState(0);
+
+    const styles = {
+        backgroundImage: `url(${CarouselData[currentImageIndex]})`,
+    };
+
+    const nextSlide = () => {
+        currentImageIndex + 1 === CarouselData.length
+            ? setCurrentImageIndex(() => 0)
+            : setCurrentImageIndex((index) => index + 1);
+    };
+
+    const previousSlide = () => {
```

Comment:
it is good that you made the slider...but i would suggest to look into react-responsive-carousel